<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogRelayConfig</class>
 <widget class="QDialog" name="LogRelayConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogRelayConfig</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QLineEdit" name="edt_passwd2">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="edt_passwd1">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Database</string>
       </property>
       <property name="buddy">
        <cstring>edt_db</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="chk_passwd">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Show Password</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edt_port">
       <property name="text">
        <string>3306</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edt_user"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="edt_db"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edt_ip"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Server Port</string>
       </property>
       <property name="buddy">
        <cstring>edt_port</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Username</string>
       </property>
       <property name="buddy">
        <cstring>edt_user</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Password</string>
       </property>
       <property name="buddy">
        <cstring>edt_passwd1</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Server IP</string>
       </property>
       <property name="buddy">
        <cstring>edt_ip</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_write">
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>edt_ip</tabstop>
  <tabstop>edt_port</tabstop>
  <tabstop>edt_user</tabstop>
  <tabstop>edt_passwd1</tabstop>
  <tabstop>edt_passwd2</tabstop>
  <tabstop>chk_passwd</tabstop>
  <tabstop>edt_db</tabstop>
  <tabstop>btn_write</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_write</sender>
   <signal>clicked()</signal>
   <receiver>LogRelayConfig</receiver>
   <slot>onBtnWrite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>LogRelayConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_passwd</sender>
   <signal>toggled(bool)</signal>
   <receiver>LogRelayConfig</receiver>
   <slot>onChkPassword(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edt_passwd2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LogRelayConfig</receiver>
   <slot>onEdtPassword2(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnWrite()</slot>
  <slot>onChkPassword(bool)</slot>
  <slot>onEdtPassword2(QString)</slot>
 </slots>
</ui>
